### ISSUE TITLE
[Feature] Visual feedback for tasks

### ISSUE THREAD

**Issue by reven** [hexparrot/mineos-node#488]
It would be great for some kind of visual feedback when updating profiles or doing other similar tasks. Using the "Refresh profile list" link in the top right drop-down always feels like I don't know if it's on-going or there was some kind of issue. I thought maybe changing the notification bell icon to a spinning wheel or similar icon would be simple way to convey "I'm working on it", without having to wonder if mineos is still fetching profiles or if I didn't click the link.

**Comment by davidbdyer** [comment]
Yes! The only hint i have that something is happening is by watching network usage in Portainer.
